zerver/tests/test_decorators.py

```diff
@@ -546,6 +546,15 @@ def test_check_none_or(self):
         x = 'x'
         self.assertEqual(check_none_or(check_int)('x', x), 'x is not an integer')
 
+    def test_check_url(self):
+        # type: () -> None
+        url = "http://127.0.0.1:5002/"  # type: Any
```

Comment:
Actually there are a lot of combinations which can be tried. But thing is I had used a django module for checking the URL validity. So I thought if the test works for one bad URL, it should work for all. What do you think?